libraries/AP_Common/Bitmask.h

```diff
@@ -23,26 +23,26 @@
 
 #include <AP_InternalError/AP_InternalError.h>
 
-template<uint16_t num_bits>
+#define NUMWORDS ((NUMBITS+31)/32)
```

Comment:
isn't this require more division calculation when previously, we store do it once and store the result into numwords?

Reply:
can we change the define to a constexpr function return to get rid of the division calculation at compile time, if that isn't done by compiler already ?

Reply:
The compiler will optimize it either way as `NUMBITS` is a constant here but it is a bit cleaner to keep it as a constant variable than a define.